### PR TITLE
JIRA-3288 specify mapreduce.job.queuename when submit sqoop job

### PR DESCRIPTION
**Pull request by Zhanshaoxiong**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: source-hive/src/main/java/org/apache/kylin/source/jdbc/JdbcHiveMRInput.java [hunk 1]
```diff
@@ -162,13 +173,16 @@ private AbstractExecutable createSqoopToFlatHiveStep(String jobWorkingDir, Strin
                 bquery += " WHERE " + partitionString;
             }
 
-            String cmd = String.format(String.format(
+            //related to "kylin.engine.mr.config-override.mapreduce.job.queuename"
+            String queueName = getSqoopJobQueueName(config);
+            String cmd = String.format(
                     "%s/sqoop import -Dorg.apache.sqoop.splitter.allow_text_splitter=true "
+                            + "- Dmapreduce.job.queuename=%s "
```
Comment (lidongsjtu) [inline source-hive/src/main/java/org/apache/kylin/source/jdbc/JdbcHiveMRInput.java]: "- Dmapreduce.job.queuename  这里，-和D中间是不是多了个空格？

**Comment by Zhanshaoxiong** [comment]
@lidongsjtu Hi Dong, please help me verify. Thanks~

**Pull request merged**